packages/workshop-app/app/components/epic-video.tsx

```diff
@@ -163,21 +163,20 @@ export function EpicVideo({
 }) {
 	return (
 		<div>
+			{muxPlaybackId ? (
+				<MuxPlayer playbackId={muxPlaybackId} />
+			) : (
+				<EpicVideoEmbed url={urlString} title={title} />
+			)}
 			{/* eslint-disable-next-line react/jsx-no-target-blank */}
 			<a
 				href={urlString}
 				target="_blank"
-				className="flex items-center gap-1 underline"
+				className="flex items-center gap-1 text-base no-underline opacity-70 transition hover:underline hover:opacity-100"
 			>
 				<Icon name="Video" size={24} title="EpicWeb.dev video" />
-				{title}
-				<Icon name="ExternalLink" title="Open video" />
+				{title} <span aria-hidden>↗︎</span>
```

Comment:
moved video links below video as they are secondary option

Reply:
Good call!

I'm also wondering if we should move the video above the list of lessons because that list can be quite long resulting in the video being below the fold. We've already got the second list at the bottom so maybe we just remove the top list?